python/private/pypi/extension.bzl

```diff
@@ -406,21 +427,25 @@ def _configure(config, *, platform, os_name, arch_name, config_settings, env = {
             # the lowest priority one needs to be the first one
             platform_tags = ["any"] + platform_tags
 
+        want_abis = want_abis or [
+            "cp{0}{1}",
+            "abi3",
```

Comment:
Why is abi3 here, but not in the MODULE config?

Reply:
Done, incorporated in #3058.